themes/addons/templates/Layout/Addon.ss

```diff
@@ -1,254 +1,190 @@
+			The detailed information for this add-on has not yet been built.
+		</p>
+	<% end_if %>
+
+	<% if $Screenshots %>
+		<ul class="screenshots inline">
+			<% loop $Screenshots %>
+				<li>
+					<a href="$URL" target="_blank">
+						<img src="$CroppedImage(160, 160).URL" alt="$Up.Name.ATT" class="img-polaroid">
+					</a>
+				</li>
+			<% end_loop %>
+		</ul>
+	<% end_if %>
+
+	<dl id="metadata">
+		<% with $MasterVersion %>
+			<dt>Homepage:</dt>
+			<dd>
+				<% if $DisplayHomepage %>
+				<a href="$DisplayHomepage" rel="nofollow">$DisplayHomepage</a>
+				<% end_if %>
+			</dd>
+
+		<% end_with %>
+
+		<dt>Packagist:</dt>
+		<dd><a href="$PackagistUrl" rel="nofollow">$PackagistUrl</a></dd>
+
+		<dt>Repository:</dt>
+		<dd>
+			$Repository
+			<object classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000"
+				width="110"
+				height="14"
+				class="clippy">
+			<param name="movie" value="$ThemeDir/thirdparty/clippy/clippy.swf"/>
+			<param name="allowScriptAccess" value="always" />
+			<param name="quality" value="high" />
+			<param name="scale" value="noscale" />
+			<param NAME="FlashVars" value="text=$Repository">
+			<param name="bgcolor" value="#fff">
+			<embed src="$ThemeDir/thirdparty/clippy/clippy.swf"
+				width="110"
+				height="14"
+				name="clippy"
+				quality="high"
+				allowScriptAccess="always"
+				type="application/x-shockwave-flash"
+				pluginspage="http://www.macromedia.com/go/getflashplayer"
+				FlashVars="text=$Repository"
+				bgcolor="#fff"
+			/>
+			</object>
+		</dd>
+
+		<% if $Authors %>
+			<dt>Authors:</dt>
+			<dd>
+				<ul id="authors">
+				<% loop $Authors %>
+					<li>
+						<a href="$Link">
+							<img src="$GravatarUrl(32)" class="img-polaroid" alt="$Name.ATT">
+							$Name
+						</a>
+					</li>
+				<% end_loop %>
+				</ul>
+			</dd>
+		<% end_if %>
+	</dl>
+
+	<hr>
+
+	<% if $Readme %>
+		<h3>Readme</h3>
+
+		<div id="readme" class="collapsed">
+			<div id="readme-inner">
+				$Readme
+			</div>
+			<a href="#" id="readme-toggle">
+				<i class="icon-arrow-up"></i>
+				<i class="icon-arrow-down"></i>
+			</a>
+		</div>
+
+		<hr>
+	<% end_if %>
+
+	<h3 id="rating-breakdown">Module rating breakdown</h3>
+
+	<div class="row">
+		<div class="span5">
+			<div class="rating rating-border">
+				<% include ModuleRatingVisual %>
+				<% if $Rating %>
+				<p>$Rating<small>/100</small></p>
+				<% else %>
+				<p>N/A <small>No data avaliable</small></p>
+				<% end_if %>
+			</div>
+			<p>Module rating system helping users find modules that are well supported. For more on how the rating system works visit <a href="http://www.silverstripe.org/software/addons/supported-modules-definition/">Module standards</a></p>
+			<p><small>Score not correct? <a href="mailto:community@silverstripe.com">Let us know there is a problem</a></small></p>
+		</div>
+
+		<% if $RatingData %>
+		<div class="span3 offset1">
+            <% include ModuleRatingEntry Metric=$RatingData.has_readme, Description="Readme" %>
```

Comment:
This really seems like it wants to be a <% loop %>

Reply:
Yeah. I'd love it to come from the config.yml in the module, but not sure what the best way to do it would be. I've been removing the check description during import since its the same for every module and would just waste DB space. For now I'll put these into a controller method and loop it instead